cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -167,10 +167,16 @@
 
        ;; Like `Move` but with a particular `PReg` source (for implementing CLIF
        ;; instructions like `get_stack_pointer`).
-       (MovPReg
+       (MovFromPReg
         (rd WritableReg)
         (rm PReg))
 
+       ;; Like `Move` but with a particular `PReg` destination (for
+       ;; implementing CLIF instructions like `get_stack_pointer`).
```

Comment:
These docs are a copy paste that should be updated for the actual use of this instruction.

Reply:
Thanks for catching that!